plotly/offline/offline.py

```diff
@@ -8,40 +8,29 @@
 import json
 import os
 import uuid
-
-import requests
+from pkg_resources import resource_string
+import warnings
 
 from plotly import session, tools, utils
 from plotly.exceptions import PlotlyError
 
-PLOTLY_OFFLINE_DIRECTORY = plotlyjs_path = os.path.expanduser(
-    os.path.join(*'~/.plotly/plotlyjs'.split('/')))
-PLOTLY_OFFLINE_BUNDLE = os.path.join(PLOTLY_OFFLINE_DIRECTORY,
-                                     'plotly-ipython-offline-bundle.js')
-
 
 __PLOTLY_OFFLINE_INITIALIZED = False
 
 
 def download_plotlyjs(download_url):
-    if not os.path.exists(PLOTLY_OFFLINE_DIRECTORY):
-        os.makedirs(PLOTLY_OFFLINE_DIRECTORY)
-
-    res = requests.get(download_url)
-    res.raise_for_status()
+    warnings.warn('''
+        `download_plotlyjs` is deprecated and will be removed in the
+        next release. plotly.js is shipped with this module, it is no
+        longer necessary to download this bundle separately.
+    ''', DeprecationWarning)
+    pass
```

Comment:
I can't really keep this functionality in here *and* include a shipped version of `plotly.js`. I can either remove this function, or keep the warning in case someone finds an old reference of this function. In either case, the purpose of this function is redundant - a `pass` or performing the download have the same affect. 
Keep the function, or keep the warning?